### ISSUE TITLE
cut off screen and D-Pad scroll not working is bug

### ISSUE THREAD

**Issue by HeyTayo74657635** [nielsvanvelzen/tv-launcher#32]
this is D-Pad left and right is not scrolling are bug
https://user-images.githubusercontent.com/99170951/193715267-00f3cd5e-44e6-46ec-9a56-c7350565c78d.mp4

and also cut off screen on top
![bandicam 2022-10-04 08-31-18-627](https://user-images.githubusercontent.com/99170951/193715389-931c5d19-d79b-424a-9d77-cb16fedd7c9b.jpg)

**Comment by nielsvanvelzen** [comment]
I'm aware of this issue but didn't have the time to fix it yet. It's caused by the recent migration to Jetpack Compose. You can try an older version of the app but it won't support channels.

**Comment by HeyTayo74657635** [comment]
visual glitch and focus column are fixed
https://user-images.githubusercontent.com/99170951/193839653-9b7b52a2-2390-4348-b889-a08d79dff79b.mp4
and press down to focus row are bug

**Comment by nielsvanvelzen** [comment]
Yeah I know. I haven't properly fixed it yet. Need to wait for proper compose support for Android TV so it will take a while. This app is not considered stable yet.
